### ISSUE TITLE
Check publications references of None

### ISSUE THREAD

**Issue by bferguso** [bcgov/BCHeritage#864]
When indexing publications, receiving 6 messages stating ` Resource with id "None" not in the system.`. Need to check why that's the case.

**Comment by bferguso** [comment]
The data loads were assuming that the following publication types were hierarchical, which I think is incorrect:

Geological Survey of Canada Bulletin
Geological Survey of Canada Current Research
Geological Survey of Canada Memoir
Geological Survey of Canada Open file
Geological Survey of Canada Paper

@LeannePyle - would you please confirm that publications of the above types are standalone publications (ie - they're not grouped like Journal Articles would be grouped in a Journal Volume)?

**Comment by LeannePyle** [comment]
Yes, these are various types of GSC publications that have different formats of "report number" rather than a sequential volume number

**Comment by bferguso** [comment]
Updated load to remove parent value from these types.

**Comment by bferguso** [comment]
Automated cleanup
